src/main/java/no/nav/pto/veilarbportefolje/domene/filtervalg/DinSituasjonSvarDto.java

```diff
@@ -0,0 +1,16 @@
+package no.nav.pto.veilarbportefolje.domene.filtervalg;
+
+public enum DinSituasjonSvarDto {
```

Comment:
I don't think it is necessary to mix the "DTO" concept into enums and their names. 
```suggestion
public enum DinSituasjonSvar {
```

Reply:
@slovrid here er requested changes: https://github.com/navikt/veilarbportefolje/pull/1126/commits/9b37f6d4fc33f6c04f531223edbaab43faa2fa4d